src/main/java/com/laytonsmith/core/functions/StringHandling.java

```diff
@@ -865,18 +865,29 @@ public Construct exec(Target t, Environment environment, Construct... args) thro
 				    args[0].typeof() + " was found.", t);
 			}
 			String text = args[0].nval();
-			
 			// Enforce the fact we are only taking the first character here
 			// Do not let the user pass an entire string. Only a single character.
 			if(text.length() > 1) {
-			    throw new CRECastException("Got " + text + " instead of expected character.", t);
+			    throw new CRECastException("Got \"" + text + "\" instead of expected character.", t);
```

Comment:
A CREFormatException would be more appropriate here, as well as the exception below. CRECastException is reserved for when, say, an int is expected, but a string is given. In this case, a string would still be given, it would just have the wrong format. See: http://wiki.sk89q.com/wiki/CommandHelper/Staged/Exceptions#FormatException